swift-sdk/Internal/EmbeddedMessagingManager.swift

```diff
@@ -143,7 +143,15 @@ class EmbeddedMessagingManager: NSObject, IterableEmbeddedMessagingManagerProtoc
                 },
                 
                 receiveError: { sendRequestError in
-                    ITBError()
+                    //TODO: This check can go away once eligibility based retrieval comes in place.
+                    if sendRequestError.reason == "SUBSCRIPTION_INACTIVE" ||
```

Comment:
We don't have the subscription inactive error ready yet but if you cut a ticket, we can certainly add that. also, i'd suggest hardcoding these error values somewhere for easier extendibility.